www/data_module/src/services/socket/socket.service.coffee

```diff
@@ -71,15 +71,20 @@ class Socket extends Service
 
             getUrl: ->
                 host = $location.host()
-                port = if $location.port() is 80 then '' else ':' + $location.port()
-                return "ws://#{host}#{port}/ws"
+                protocol = if $location.protocol() is 'https' then 'wss' else 'ws'
+                defaultport = if $location.protocol() is 'https' then 443 else 80
+                path = $location.path()
```

Comment:
$location.path() return 'bar' when url is 'http://some.com/foo/#/bar
while we want /foo/. See last version of my PR